includes/wikia/services/TemplateClassificationService.class.php

```diff
@@ -45,22 +46,25 @@ class TemplateClassificationService {
 	 * @throws \Swagger\Client\ApiException
 	 */
 	public function getType( $wikiId, $pageId ) {
-		$templateType = self::TEMPLATE_UNCLASSIFIED;
-
-		try {
-			$type = $this->getApiClient()->getTemplateType( $wikiId, $pageId );
-			if ( !is_null( $type ) ) {
-				$templateType = $type->getType();
+		if ( !isset( $this->cache[$wikiId][$pageId] ) ) {
+			$templateType = self::TEMPLATE_UNCLASSIFIED;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)